code/modules/jobs/job_types/_job.dm

```diff
@@ -440,7 +443,7 @@
 	if(!player_client)
 		return // Disconnected while checking for the appearance ban.
 
-	var/require_human = CONFIG_GET(flag/enforce_human_authority) && (job.departments_bitflags & DEPARTMENT_BITFLAG_COMMAND)
+	var/require_human = !job.ignore_human_authority && CONFIG_GET(flag/enforce_human_authority) && (job.departments_bitflags & DEPARTMENT_BITFLAG_COMMAND)
```

Comment:
If QM is being a head, they shouldn't get some special exemption to being non-human

Reply:
i told goof they should because i like lizard qms

Reply:
waiting on oranges response to this pr

Reply:
The deep lore spinsters union is great stuff though.